### PR TITLE
[Merged by Bors] - feat: Dot product and cross product on projective space

### PR DESCRIPTION
**Pull request by tb65536**
This PR defines the dot product and cross product on projective space. This will be used to define finite projective planes.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by jcommelin — COMMENTED** [review]
Sorry for the delayed review....

File: Mathlib/LinearAlgebra/Projectivization/Constructions.lean [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+variable {F : Type*} [Field F] {m : Type*} [Fintype m]
+
+namespace Projectivization
+
+open scoped LinearAlgebra.Projectivization
+
+section DotProduct
+
+/-- Orthogonality on the projective plane. -/
+def orthogonal : ℙ F (m → F) → ℙ F (m → F) → Prop :=
```
[the remark this answers is not in the document]
Comment (tb65536) [reply Mathlib/LinearAlgebra/Projectivization/Constructions.lean:33]: Mathlib's definition of inner product space assumes `RCLike`, so the definition would not work for finite fields which is the ultimate goal here.

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Comment by jcommelin** [comment]
bors d+

**Pull request closed without merging**